8-writing-advanced-tasks/README.md

```diff
@@ -4,251 +4,258 @@
 
 > **Time**: Approximately 10 minutes
 
-In this exercise you will write a task with metadata. Including a task that supports_noop.
+In this exercise you will write a task with metadata. 
 
 # Prerequisites
-
-For the following exercises you should already have `bolt` installed and have a few nodes (either Windows or Linux) available to run commands against. The following guides will help:
+Complete the following before you start this lesson:
 
 1. [Installing Bolt](../1-installing-bolt)
 1. [Acquiring nodes](../2-acquiring-nodes)
 1. [Writing tasks](../5-writing-tasks)
 
-# Writing your first task with metadata
+# About task metadata
+Task metadata files describe task parameters, validate input, and control tasks are executed.  Adding metadata to your tasks helps others use them.  You write metadata for a task in JSON and save it with the same name as your task. For example, if you write a task called `great_metadata.py` its corresponding metadata file is named `great_metadata.json`.  
```

Comment:
should be be "control how tasks are executed"

Reply:
fixed